packages/expo-linking/src/__tests__/Linking-test.ts

```diff
@@ -1,4 +1,5 @@
 import Constants from 'expo-constants';
+import { ExecutionEnvironment } from 'expo-constants/build/Constants.types';
```

Comment:
should we export this time from expo-constants so it can be used eg: in the Linking.ts code above?